test/LoginStrategy.siwetoken.test.ts

```diff
@@ -131,6 +132,45 @@ it('Should authenticate issuer signature with Siwe', async () => {
   });
 });
 
+it('Should authenticate issuer signature with Siwe for ewc chain', async () => {
+  const { loginStrategy } = preparePassport(
+    provider,
+    ensResolver.address,
+    didContract.address,
+    ensRegistry.address
+  );
+  const sampleSiwePayloadWithEwcChainID = new SiweMessagePayload({
+    domain: 'login.xyz',
+    address: wallet.address,
+    statement: 'Sign-In With Ethereum Example Statement',
+    uri: 'https://login.xyz',
+    version: '1',
+    nonce: 'bTyXgcQxn2htgkjJn',
+    issuedAt: '2022-01-27T17:09:38.578Z',
+    chainId: 246,
+    expirationTime: '2100-01-07T14:31:43.952Z',
+  });
+  const signature = await wallet.signMessage(
+    sampleSiwePayloadWithEwcChainID.prepareMessage()
+  );
+  expect(token).toBeTruthy();
```

Comment:
Where is `token` from? Is it relevant here?

Reply:
My bad, will update this.